rq/registry.py

```diff
@@ -202,6 +210,7 @@ class StartedJobRegistry(BaseRegistry):
     """
 
     key_template = 'rq:wip:{0}'
+    death_penalty_class = UnixSignalDeathPenalty
 
     def cleanup(self, timestamp: Optional[float] = None):
         """Remove expired jobs from registry and add them to FailedJobRegistry.
```

Comment:
Maybe change this to abandoned too? Then we've got the same naming everything 🤩

Reply:
Done!